config/settings.yml

```
@@ -108,6 +108,13 @@ data_map_dict:
     SOFRINDEX:
         data_ref: "SOFRINDEX"
         data_source: "FRED"
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)